### PR TITLE
docs: optimism updates and version update

### PR DESCRIPTION
**Pull request by jcstein**
## Summary by CodeRabbit

- **Documentation**
  - Updated the integration guide for OP Stack with Celestia, emphasizing Celestia's role as a data availability network to enhance scalability.
  - Revised terminology and descriptions in the documentation to align with current functionalities and tools involved in data handling.

### CODE REVIEW

**Comment by tuxcanfly** [comment]
Since we're switching away from celestia-da, we can replace `grpcurl` commands with `curl` equivalents:

First we need the `CELESTIA_NODE_AUTH_TOKEN` to call the node API, e.g. for mocha:

`export CELESTIA_NODE_AUTH_TOKEN=$(celestia light auth admin --node.store /home/celestia/.celestia-light-mocha-4)`

    grpcurl -proto da.proto -plaintext -d '{"blobs": [{"value": "SGVsbG8gd28ybGQh"}]}' 127.0.0.1:26650 da.DAService.Submit

can be replaced with:

    curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Submit", "params": [["SGVsbG8gd28ybGQh"], -1, "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658

which should return:

> {"jsonrpc":"2.0","result":["9QEAAAAAAABmGvJUORYLqY8wVdjgp6/0oSh6brQqG3ZGr/cfUMuElg=="],"id":1}

similarly

    grpcurl -proto da.proto -plaintext -d "{\"ids\": [{\"value\": \"$ENCODED_INPUT\"}]}" 127.0.0.1:26650 da.DAService.Get


can be replaced with:

    curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Get", "params": [["3wAAAAAAAABmGvJUORYLqY8wVdjgp6/0oSh6brQqG3ZGr/cfUMuElg=="], "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658
    
which should return:

> {"jsonrpc":"2.0","result":["SGVsbG8gd28ybGQh"],"id":1}

**Comment by jcstein** [comment]
where should `$ENCODED_INPUT` be in: `curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Get", "params": [["3wAAAAAAAABmGvJUORYLqY8wVdjgp6/0oSh6brQqG3ZGr/cfUMuElg=="], "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658`?

File: developers/optimism.md [hunk 1]
```diff
@@ -371,34 +356,16 @@ Remember to remove the `0xce` prefix!
...
-cd go-da/proto/da
-```
-
-Now, from `go-da/proto/da` run:
-
 <!-- markdownlint-disable MD013 -->
 
 ```bash
-grpcurl -proto da.proto -plaintext -d "{\"ids\": [{\"value\": \"$ENCODED_INPUT\"}]}" 127.0.0.1:26650 da.DAService.Get
+curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Get", "params": [["3wAAAAAAAABmGvJUORYLqY8wVdjgp6/0oSh6brQqG3ZGr/cfUMuElg=="], "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658
```
Comment (jcstein) [inline developers/optimism.md:362]: this doesn't work for me and i'll need to revisit tomorrow

File: developers/optimism.md [hunk 2]
```diff
@@ -480,24 +447,26 @@ Celestia.
...
 cause the `op-batcher` blob transaction to fail, with an `incorrect account
 sequence` error, which triggers a fallback to Ethereum.
 
-To trigger the transaction, send this command from the same `go/proto/da` directory:
-
 <!-- markdownlint-disable MD013 -->
 
 ```bash
-grpcurl -proto da.proto -plaintext -d '{"blobs": [{"value": "SGVsbG8gd28ybGQh"}]}' 127.0.0.1:26650 da.DAService.Submit
+curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Submit", "params": [["SGVsbG8gd28ybGQh"], -1, "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658
```
Comment (jcstein) [inline developers/optimism.md:458]: same for this

**Comment by tuxcanfly** [comment]
> where should `$ENCODED_INPUT` be in: `curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Get", "params": [["3wAAAAAAAABmGvJUORYLqY8wVdjgp6/0oSh6brQqG3ZGr/cfUMuElg=="], "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658`?

should be:

    curl -s -X POST -H "Content-Type: application/json" -H "Authorization: Bearer $CELESTIA_NODE_AUTH_TOKEN" -d '{ "id": 1, "jsonrpc": "2.0", "method": "da.Get", "params": [["$ENCODED_INPUT"], "AAAAAAAAAAAAAAAAAAAAAAAAAAECAwQFBgcICRA="]}' http://127.0.0.1:26658

the params are `[]blobs, namespace`, base64-encoded.

File: developers/optimism.md [hunk 3]
```diff
@@ -86,76 +115,85 @@ $HOME/.celestia-light-{{constants.arabicaChainId}}
...
-  image: ghcr.io/rollkit/celestia-da:v0.12.9 // [!code ++]
-  command: > // [!code ++]
-    celestia-da light start // [!code ++]
-    --p2p.network=<network> // [!code ++]
-    --da.grpc.namespace=000008e5f679bf7116cb // [!code ++]
-    --da.grpc.listen=0.0.0.0:26650 // [!code ++]
-    --core.ip <rpc-url> // [!code ++]
-    --gateway // [!code ++]
+  image: ghcr.io/rollkit/local-celestia-devnet:v0.13.1 // [!code --]
+  image: ghcr.io/celestiaorg/celestia-node:v0.13.2 // [!code ++]
```
Comment (tuxcanfly) [inline developers/optimism.md:164]:
```suggestion
  image: ghcr.io/celestiaorg/celestia-node:v0.13.2 // [!code ++]
```

File: developers/optimism.md [hunk 4]
```diff
@@ -199,3 +264,219 @@ make devnet-clean
...
+
+These two functions work together to ensure that the Ethereum
+fallback mechanism operates correctly, allowing the rollup
+to continue functioning even during periods of downtime on
+Celestia.
+
+### Testing the fallback
+
+Testing out the Ethereum fallback mechanism can be done
+with the `go-da` tool. Triggering a simultaneous blob transaction will
```
Comment (tuxcanfly) [inline developers/optimism.md:451]:
```suggestion
Testing out the Ethereum fallback mechanism can be done
with the a curl command. Triggering a simultaneous blob transaction will
```

**Review by tuxcanfly — APPROVED** [review]
LGTM, with minor changes.

File: developers/optimism.md (hunk 1 above)
Comment (tuxcanfly) [reply developers/optimism.md:362]: Hmm, what is the output?

**Comment by tuxcanfly** [comment]
Released https://github.com/celestiaorg/optimism/releases/tag/v1.3.0-OP_op-node%2Fv1.7.5-CN_v0.13.2 which includes the new setup.

File: developers/optimism.md (hunk 3 above)
Comment (jcstein) [reply developers/optimism.md:164]: https://github.com/celestiaorg/docs/pull/1566/commits/b8681cd21f15cdb2ab311ac42ffe98de91bf05ff

File: developers/optimism.md (hunk 4 above)
Comment (jcstein) [reply developers/optimism.md]: https://github.com/celestiaorg/docs/pull/1566/commits/b8681cd21f15cdb2ab311ac42ffe98de91bf05ff

**Pull request merged**